imagebuildah/build.go

```diff
@@ -253,6 +253,23 @@ func BuildDockerfiles(ctx context.Context, store storage.Store, options define.B
 		}
 		platformOptions.Args = argsCopy
 		builds.Go(func() error {
+			if platformOptions.LogFile != "" && platformOptions.LogSplitByPlatform {
+				logFile := platformOptions.LogFile + "_" + platformOptions.OS + "_" + platformOptions.Architecture
+				f, err := os.OpenFile(logFile, os.O_CREATE|os.O_TRUNC|os.O_WRONLY, 0600)
+				if err != nil {
+					return errors.Errorf("error opening logfile %q: %v", logFile, err)
+				}
+				defer f.Close()
+				logger = logrus.New()
```

Comment:
Isn't `logger` shared between all of these goroutines?

Reply:
In this case since logger for all the goroutine is writing to their own unique filepaths hence I thought its better to re-init since for this particular case sharing was not helping a lot since all the logs go to different files for each go-routine.

Reply:
Should be fixed now.